guides/source/configuring.md

```diff
@@ -1481,8 +1481,12 @@ queries can be executed concurrently.
 
 Defaults to `4`.
 
-This number must be considered in accordance with the database pool size configured in `database.yml`. The connection pool
-should be large enough to accommodate both the foreground threads (.e.g web server or job worker threads) and background threads.
+This number must be considered in accordance with the database connection pool size configured in `database.yml`. The connection pool
+should be large enough to accommodate both the foreground threads (ie. web server or job worker threads) and background threads.
+
+For each process, Rails will create one global query executor that uses this many threads to process async queries. Thus, the pool size
+must be at least `(thread_count + 1) + global_executor_concurrency`. For example, if your web server has a maximum of 3 threads,
```

Comment:
I know I only wrote this yesterday, but I can't remember what the `+ 1` is for... 🤔

Reply:
I assume that's the main/root/control thread. You really shouldn't be running queries there, but people do -- generally by having models that introspect the database at class load time. 

Even still, it should be moderately difficult to have it still hold a connection by the time you're accepting work... but if you're aiming to describe the full count of threads that will exist and could _possibly_ make use of a connection in a standard application [i.e., one that does not have any other 3rd-party threading going on], it's more technically correct to include it.

Now there's no multiplication going on, the parentheses are just confusing, though.

(IMO "must" is overly strong, at least without an accompanying "to avoid any contention" or similar -- _especially_ if we're adding one for luck, per above.)

Reply:
Yeah, I think I must have been thinking of a control thread. But I don't remember explicitly thinking about that. Aging sucks.

- Changed "must" to "should".
- Removed the brackets.
- I don't mind keeping the `+1` if it's more confusing than helpful, either way I'm pretty happy with this.